tests/unit/frameworks/pytorch/restore_utils_test.py

```diff
@@ -212,10 +220,13 @@ def test_load_nonlinear(self):
         num_matches = out.isclose(self.out_lower, atol=1e-2).sum().item()
         self.assertEqual(num_matches, 2048)  # all correct
 
-        # Check output through the lower network.
-        out = upper_forward(model, self.in_2)
-        num_matches = out.isclose(self.out_upper, atol=1e-2).sum().item()
-        self.assertEqual(num_matches, 1)
+        # These exact numbers are random, the test should test for something
```

Comment:
Instead of commenting out the test could you add a "skip" comment just like you added to `test_checkpoint_backward_compatibility`?